src/WrapperApp/components/InputEditor/InputFilesEditor.tsx

```diff
@@ -37,6 +37,20 @@ export function InputFilesEditor(props: InputFilesEditorProps) {
 	const largeFileSize = 100_000;
 	const largeFileLinesLimit = 500;
 
+	// const estimatorNames = [];
```

Comment:
that won't work for Fluka and some special types of SHIELD-HIT12A input files.
Why not having a dedicated Flask endpoint for that ?